### ISSUE TITLE
Installing pyICU fails on macOS

### ISSUE THREAD

**Issue by philschatz** [openstax/cnx-easybake#82]
I tried installing using #76 and it failed.

According to https://tobywf.com/2017/05/installing-pyicu-on-macos/ it seems like the setup script needs to do the following:

```sh
$ brew install icu4c
[...]
$ ls /usr/local/Cellar/icu4c/
58.2
$ export ICU_VERSION=58
$ export PYICU_INCLUDES=/usr/local/Cellar/icu4c/58.2/include
$ export PYICU_LFLAGS=-L/usr/local/Cellar/icu4c/58.2/lib
```

The same solution would also need to be added to https://github.com/Connexions/cnx-recipes

/cc @reedstrm @helenemccarron 

# Stack Trace

<details>

<summary>Click me to see the Stack Trace</summary>

```
Installed /path/to/cnx-easybake/venv/lib/python2.7/site-packages/cnx_easybake-1.0.0+4.gb6ff07b-py2.7.egg
Processing dependencies for cnx-easybake==1.0.0+4.gb6ff07b
Searching for PyICU
Reading https://pypi.python.org/simple/PyICU/
Downloading https://pypi.python.org/packages/17/13/12b5631b67fd27b8827fb805df5015c32406ec739d700af68ab49da1203f/PyICU-1.9.8.tar.gz#md5=010b1e2e44324b41ab2c1a77dbec63e2
Best match: PyICU 1.9.8
Processing PyICU-1.9.8.tar.gz
Writing /var/folders/dn/czks72854yvb6b1lq46gxqqc0000gn/T/easy_install-AqqImJ/PyICU-1.9.8/setup.cfg
Running PyICU-1.9.8/setup.py -q bdist_egg --dist-dir /var/folders/dn/czks72854yvb6b1lq46gxqqc0000gn/T/easy_install-AqqImJ/PyICU-1.9.8/egg-dist-tmp-kyyHYm
zip_safe flag not set; analyzing archive contents...
cnxeasybake.tests.test_cli: module references __file__
cnxeasybake.tests.test_rulesets: module references __file__
Traceback (most recent call last):
  File "setup.py", line 44, in <module>
    'git+https://github.com/Connexions/cssselect2.git#egg=cssselect2']
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/install.py", line 117, in do_egg_install
    cmd.run()
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 412, in run
    self.easy_install(spec, not self.no_deps)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 654, in easy_install
    return self.install_item(None, spec, tmpdir, deps, True)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 701, in install_item
    self.process_distribution(spec, dist, deps)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 746, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 866, in resolve
    replace_conflicting=replace_conflicting
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1146, in best_match
    return self.obtain(req, installer)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1158, in obtain
    return installer(requirement)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 673, in easy_install
    return self.install_item(spec, dist.location, tmpdir, deps)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 699, in install_item
    dists = self.install_eggs(spec, download, tmpdir)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 882, in install_eggs
    return self.build_and_install(setup_script, setup_base)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 1150, in build_and_install
    self.run_setup(setup_script, setup_base, args)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 1136, in run_setup
    run_setup(setup_script, args)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 253, in run_setup
    raise
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 195, in setup_context
    yield
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 166, in save_modules
    saved_exc.resume()
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 141, in resume
    six.reraise(type, exc, self._tb)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 154, in save_modules
    yield saved
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 195, in setup_context
    yield
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 250, in run_setup
    _execfile(setup_script, ns)
  File "/path/to/cnx-easybake/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 45, in _execfile
    exec(code, globals, locals)
  File "/var/folders/dn/czks72854yvb6b1lq46gxqqc0000gn/T/easy_install-AqqImJ/PyICU-1.9.8/setup.py", line 33, in <module>
    tests_require=tests_require,
RuntimeError: 
Please set the ICU_VERSION environment variable to the version of
ICU you have installed.
        
./script/setup: ERROR: could not run [python setup.py install]
```

</details>

**Comment by philschatz** [comment]
Now fixed in #76 😄

**Comment by brittweinstein** [comment]
CLOSING. If this presents itself as an issue again, we will reopen.
